db/seeds.rb

```diff
@@ -5,3 +5,8 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
+
+category = Category.create!(name: "test")
```

Comment:
how can you make it so this seed is re-runnable? Also would you realistically seed data like this or should you create some semi-real categories and products?